### DISCUSSION TITLE
Silently install plugins

### DISCUSSION THREAD

**Discussion by Riki7745** [QL-Win/QuickLook#1277] [Ideas]
All is in the title, could it be possible to install plugins silently with command lines...?
Thanx in advance.

**Comment by xupefei** [comment]
It's possible: `unzip` the plugin package to the correct [data location](https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions#user-data-location).

**Reply by Riki7745** [reply]
> It's possible: `unzip` the plugin package to the correct [data location](https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions#user-data-location).

Yes, I saw that, but I thought about a more..."conventional" way of doing it, like "quicklook.exe -installplugin -plugin -QuickLook.Plugin.FolderViewer.qlplugin" for example...
